frontend/src/pages/events/prayerGatheringEvents.js

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+
+const PrayerGatheringEvents = () => {
+  return (
+    <div className="bg-Neutral-200 w-full justify-center flex flex-col items-center py-[40px]">
+      <div className="flex-col justify-start items-center gap-y-2 flex mt-2">
+        <div className="subheading">PRAYER GATHERINGS</div>
+        <div className="text-black text-4xl font-semibold leading-10">BY PRAYING, WE LEARN TO PRAY.</div>
+      </div>      
+      <div className="text-xl sm:text-base sm:w-2/5 px-4 pt-4 text-center text-black font-normal leading-normal">“The harvest is plentiful, but the laborers are few; therefore PRAY earnestly to the Lord of the harvest to send out laborers into his harvest.” -Matthew 9:37b-38<br/><br/>Join us for a church-wide Prayer Gathering at our Transformation Center!</div>
+      
+      <div className="mt-8 sm:mt-4 w-4/5 sm:w-1/3 sm:px-14 py-6 mb-4 bg-Shades-0 bg-opacity-80 rounded-2xl border-2 border-cyan-700 justify-center sm:justify-start items-start gap-1 sm:gap-2 inline-flex">
```

Comment:
```suggestion
     <div className="mt-8 sm:mt-4 max-w-[32.5rem] sm:px-[3.625rem] py-6 mb-4 bg-Shades-0 bg-opacity-80 rounded-2xl border-2 border-cyan-700 justify-center gap-1 sm:gap-2">
```
This is some of suggestions changes for dealing with 4k monitors which I have.